org.eclipse.xtext.tests/testdata/wizard-expectations/gradle/gradle.parent/gradle/build.gradle

```diff
@@ -21,14 +22,17 @@ dependencies {
 task generateXtextLanguage(type: JavaExec) {
 	main = 'org.eclipse.emf.mwe2.launch.runtime.Mwe2Launcher'
 	classpath = configurations.mwe2
-	inputs.file "src/main/java/org/xtext/example/mydsl/GenerateMyDsl.mwe2"
-	inputs.file "src/main/java/org/xtext/example/mydsl/MyDsl.xtext"
+	inputs.file "src/main/java/mydsl/GenerateMyDsl.mwe2"
+	inputs.file "src/main/java/mydsl/MyDsl.xtext"
 	outputs.dir "src/main/xtext-gen"
-	args += "src/main/java/org/xtext/example/mydsl/GenerateMyDsl.mwe2"
+	args += "src/main/java/mydsl/GenerateMyDsl.mwe2"
 	args += "-p"
 	args += "rootPath=/${projectDir}/.."
 }
 
+test {
```

Comment:
Change to Junit5 intentional?

Reply:
yes. wanted to have a test for that one too. see issue